packages/connectors/tsup.config.js

```diff
@@ -7,6 +7,7 @@ export default defineConfig((options) => ({
     'src/index.ts',
     'src/walletconnect-connector/index.ts',
     'src/evm-connector/index.ts',
+    'src/bako-safe/index.ts',
```

Comment:
you don't need to add bako here, this is only added because `walletconnect` and `evm` connectors are not exported in the default bundle. only the `src/index.ts` should handle bako as well